EIPS/EIP-1776.md

```diff
@@ -0,0 +1,187 @@
+---
+eip: 1776
```

Comment:
EIP-1776 is already taken : https://github.com/ethereum/EIPs/issues/1776

Reply:
Thanks, dang I searched and thought 1776 was free, yours is not in EIPs or a PR so that is why I did not see it, sorry about that. Are you going to do a PR for it? 

Will adjust over to a new value for the EIP

Reply:
yes, still planning to work on it